### DISCUSSION TITLE
Custom Icons Not Overriding Default Folder Icons

### DISCUSSION THREAD

**Discussion by realKAILEENO** [lsd-rs/lsd#1006] [Q&A] [answered]
Hello lsd maintainers and community,

I've encountered an issue where the custom icons defined in my icons.yaml file are not overriding the default icons for folders. Below are the details of my configuration and the steps I've taken to troubleshoot:
lsd Version: lsd 1.0.0
Operating System: macOS Sonoma 14.2.1
Terminal Emulator: iTerm2
Font Being Used: MesloLGS NF
Content of `icons.yaml`:

```
name:
  Developer: 󰵮
  School: 󰑴
  Movies: 󰟞
  Library: 
  Applications: 
  ```
  
Troubleshooting Steps:

1. Confirmed that `icons.yaml` is located at `~/.config/lsd/icons.yaml`.
1. Verified that my terminal supports the UTF-8 characters and emojis.
1. Checked that my terminal emulator is capable of rendering the icons.
1. Ensured there are no conflicts with other configuration files.
1. Tried different icons and terminals but observed the same issue.
1. Tried using the UTF codes of each icon.

Despite the above, the default icons remain unchanged. I would appreciate any guidance on this issue or any steps I might have missed.

Thank you for your support!

**Comment by zwpaper** [comment, marked as answer]
please try to use the lowercase dir name, looks like there is some enhancement about the icon yaml config

**Reply by realKAILEENO** [reply]
Thanks for your help! I tried using a lowercase directory name as you suggested, and it's working now.
